raddb/mods-available/dhcpv4

```diff
@@ -28,7 +28,7 @@
 #
 
 #
-#  # Default instance
+#  ## Default instance
```

Comment:
We should decide if it's `==` or `##`, and use one version everywhere.

Reply:
yes, I noticed that the `#` is a _markdown_ stuff. I will update to use the `=` levels.

Reply:
There are still a lot of `##` instead of `==`

Reply:
Maybe update the `==` as a separate commit?